src/Engine/GameVersion.java

```diff
@@ -12,11 +12,12 @@
   int majorRev, minorRev, hotfix;
   public GameVersion()
   {
-    majorRev = 0;
-    minorRev = 1;
-    hotfix = 0;
+    majorRev = 3; // For substantial, save-breaking changes to the game logic.
+    minorRev = 0; // For added content or UI updates
+    hotfix   = 0; // For bugfixes (hopefully to never see increment)
   }
   
+  // TODO: Consider more nuanced validation here
```

Comment:
We should probably sort that out before merging, since the point of this class is that it isn't supposed to change.

Reply:
Eh, not really. The class is stored as 3 ints, and the serial version is set.
Unless we actually change how many numbers we're keeping track of, we can do whatever we want without affecting serialization compatibility.

Reply:
Sounds good then.